backend/src/resolvers/users.spec.js

```diff
@@ -71,7 +77,8 @@ describe('userMiddleware', () => {
 
     it('with too short name', async () => {
       const variables = {
-        id: 'u1'
+        id: 'u47',
```

Comment:
I think I have to give myself a "Request Changes" review here. I told @ulfgebhardt not to create test cases (== specification) for behaviour that we actually don't want to support, e.g. passing `id` from the client. I'll update that as soon as possible.